genesyscloud/knowledge_document/genesyscloud_knowledge_document_init_test.go

```diff
@@ -28,7 +28,7 @@ func (r *registerTestInstance) registerTestResources() {
 	defer r.resourceMapMutex.Unlock()
 	providerResources["genesyscloud_knowledge_category"] = gcloud.ResourceKnowledgeCategory()
 	providerResources["genesyscloud_knowledge_knowledgebase"] = gcloud.ResourceKnowledgeKnowledgebase()
-	providerResources["genesyscloud_knowledge_label"] = gcloud.ResourceKnowledgeLabel()
+	providerResources["genesyscloud_knowledge_label"] = knowledgeLabel.ResourceKnowledgeLabel()
```

Comment:
You can now do `providerResources[knowledgeLabel.ResourceType] = knowledgeLabel.ResourceKnowledgeLabel()`